docs/streams/developer-guide.html

```diff
@@ -466,7 +466,419 @@
     GlobalKTable&lt;String, GenericRecord&gt; source2 = builder.globalTable("topic4", "globalStoreName");
     </pre>
 
-    <h4><a id="streams_dsl_windowing" href="#streams_dsl_windowing">Windowing a stream</a></h4>
+    <h4><a id="streams_dsl_transform" href="#streams_dsl_transform">Transform a stream</a></h4>
+
+    <code>KStream</code> and <code>KTable</code> support a variety of transformation operations. Each of these operations can be translated into one or more
```

Comment:
I know this is a bit annoying, but can we try and put each sentence on a new line? It just makes it easier to review in the future.